audio/audacity/Portfile

```diff
@@ -68,6 +68,9 @@ subport ${name} {
     post-destroot {
         platform darwin {
             move ${destroot}${prefix}/Audacity.app ${destroot}${applications_dir}/
+            if {${os.major} >= 22} {
+                system -W ${destroot}${applications_dir} "/usr/bin/codesign --sign - Audacity.app"
```

Comment:
This is just the same as we do when we add "-" to the xcodeprojects, right? My CLI signing is rusty.

Reply:
Yea this is the same as `CODE_SIGN_IDENTITY=-` or “Sign to run locally” in Xcode

Reply:
Cool! Thanks.